bitsail-connectors/bitsail-connectors-legacy/bitsail-connector-hive/src/main/java/com/bytedance/bitsail/connector/legacy/hive/sink/HiveOutputFormat.java

```diff
@@ -397,6 +403,15 @@ public void initPlugin() throws TException, IOException {
       hiveSerdeParameter = HiveMetaClientUtil.getSerdeParameters(hiveConf, db, table);
       columnMapping = HiveMetaClientUtil.getColumnMapping(hiveTableSchema);
       LOG.info("fetch column mapping from hive metastore:\n {}", columnMapping);
+      if (Objects.isNull(columns)) {
+        columns = Lists.newArrayList();
+        String[] fieldNames = hiveTableSchema.getFirst().split(",");
+        String[] fieldTypes = hiveTableSchema.getSecond().split(":");
+        for (int i = 0; i < fieldNames.length; i++) {
+          columns.add(new ColumnInfo(fieldNames[i], fieldTypes[i]));
+        }
+        LOG.info("Use columns from hive metadata: {}.", columns);
+      }
```

Comment:
when i use `HiveMetaClientUtil.getColumnInfo` to get metadata, it directly return `[]`, it's weird, and i can't see the log, so I use the current way to handle, can you help take a look, @BlockLiu thx.

```java
  if (Objects.isNull(columns)) {
    columns = HiveMetaClientUtil.getColumnInfo(hiveConf, db, table);
    LOG.info("Use columns from hive metadata: {}.", columns);
  }
```

Reply:
These two methods are both ok. 
I'm suprised that the method in comment didn't work, because they actually run in the same way.
I test this method on my laptop, and it works fine.
```java
@Test
public void test() {
    String db = "test_db";
    String table = "test_table";
    Map<String, String> hiveProperties = ImmutableMap.of(
        "hive.metastore.uris", "thrift://xx.xx.xx.xx:9083"
    );
    HiveConf hiveConf = HiveMetaClientUtil.getHiveConf(hiveProperties);

    List<ColumnInfo> columnInfoList = HiveMetaClientUtil.getColumnInfo(hiveConf, db, table);
    System.out.println(columnInfoList);
  }
```
<img width="855" alt="截屏2023-02-07 10 43 18" src="https://user-images.githubusercontent.com/22468412/217134530-76a9a6a2-e51e-4d32-8894-b21f6b13b8d1.png">